tests/identities/IdentitiesManager.test.ts

```diff
@@ -32,10 +33,12 @@ describe('IdentitiesManager', () => {
       path.join(os.tmpdir(), 'polykey-test-'),
     );
     const keysPath = `${dataDir}/keys`;
+    const keyPairOverride = await generateKeyPair(4096);
     keyManager = await KeyManager.createKeyManager({
       password,
       keysPath,
       logger,
+      keyPairOverride,
```

Comment:
Need to to review this new key generation system. The old system without the recovery key generation was faster than the new way where we are always generating with a recovery key. I want to see if that means we should be using webcrypto immediately or stick with node-forge. Plus what an override here means if there's a better way to do this instead.

Reply:
> 

`TypeError: Status is not a class`

When printing out what Status was just before the error I was getting the contents of `Status.json`.